### PR TITLE
add rudimentary doc on UInt

### PR DESCRIPTION
**Pull request by tbrowder**

### CODE REVIEW

File: doc/Type/UInt.pod6 [hunk 1]
```diff
@@ -0,0 +1,12 @@
+=begin pod
+
+=TITLE class UInt
```
Comment (zoffixznet) [inline doc/Type/UInt.pod6]: This probably should be `subset` not `class`. You can't instantiate a subset, for example, but you can a class.

Comment (tbrowder) [inline doc/Type/UInt.pod6]: On Wed, Aug 31, 2016 at 10:11 AM, Zoffix Znet <notifications@github.com> wrote:
> In doc/Type/UInt.pod6:
>
> > @@ -0,0 +1,12 @@
> > +=begin pod
> > +
> > +=TITLE class UInt
>
> This probably should be subset not class. You can't instantiate a subset, for example, but you can a class.

Hm, on #perl6 I did:

  m: my UInt $u; ...

and timotimo did, too, with no problems.

Comment (zoffixznet) [inline doc/Type/UInt.pod6]: I meant this:

> Elfoonto 	m: my $int = Int.new: 42;
> camelia 	rakudo-moar af4c2e: ( no output )
> Elfoonto 	m: my $uint = UInt.new: 42;
>  camelia 	rakudo-moar af4c2e: OUTPUT«You cannot create an instance of this type (UInt)␤  in block <unit> at <tmp> line 1␤␤»

And this:

> Elfoonto 	m: class BetterInt is Int {};
> camelia 	rakudo-moar af4c2e: ( no output )
> Elfoonto 	m: class BetterInt is UInt {};
> camelia 	rakudo-moar af4c2e: OUTPUT«5===SORRY!5=== Error while compiling <tmp>␤UInt does not support inheritance, so BetterInt cannot inherit from it␤at <tmp>:1␤»

Comment (tbrowder) [inline doc/Type/UInt.pod6]: On Wed, Aug 31, 2016 at 12:40 PM, Zoffix Znet <notifications@github.com> wrote:
>
> In doc/Type/UInt.pod6:
>
> > @@ -0,0 +1,12 @@
> > +=begin pod
> > +
> > +=TITLE class UInt
>
> I meant this:
>
> Elfoonto m: my $int = Int.new: 42;

Later I figured that out.  I threw out a challenge to either correct
the doc or move the text but keep 'UInt' searchable.

**Pull request merged**